### DISCUSSION TITLE
Connecting Multiple UEs in a 5G Network

### DISCUSSION THREAD

**Discussion by rolandmartinson7890** [srsran/srsRAN_Project#193] [General Help]
Hello srsRAN users.

I have a Google Pixel and Huawei P40. I was able to connect them but the Huawei is not stable getting kicked out and then I am getting this message from the Open5gs Core Network Log:

08/30 15:20:07.770: [gmm] WARNING: [imsi-001010123456790] Paging failed. Stop (../src/amf/gmm-sm.c:490)

Is there a known code I can dial on the Huawei to do away with this paging problem on the Huawei?

The Huawei is connecting but is not stable. The whole time it can see and is connected to the network but just isnt stable with getting in internet connection.

**Comment by robertfalkenberg** [comment]
Have you tried `*#*#0702#*#*` as described [here](https://github.com/srsran/srsRAN_Project/discussions/100)?
Other options were suggested [here](https://github.com/srsran/srsRAN_Project/discussions/73#discussioncomment-6563885), however we haven't tried that.
